### ISSUE TITLE
PHP 7.2 issues

### ISSUE THREAD

**Issue by linaspasv** [WhichBrowser/Parser-PHP#87]
Hello, 
I was trying to run the parser on PHP 7.2 with the following:

```php
$result = new WhichBrowser\Parser($_SERVER['HTTP_USER_AGENT']);
$result->toArray();
```

and received an error:
![image](https://user-images.githubusercontent.com/1407067/33483419-6a435f46-d6a6-11e7-9bf4-c6edd5935c7a.png)

Best,
Linas

**Comment by NielsLeenheer** [comment]
I can replicate the error. Will investigate.

**Comment by NielsLeenheer** [comment]
Problem found. The function `count()` was changed in PHP 7.2 in a way that is not backwards compatible.

**Comment by NielsLeenheer** [comment]
Version 2.0.28 has been released with PHP 7.2 compatibility.
